### PR TITLE
Remove the buttons and add as links in navbar

### LINKED ISSUE #221: Remove the buttons from the center of the page and have them as links on the navbar
Remove the buttons from the center of the page and have them as links on the navbar

### PR DESCRIPTION
**Pull request by subhangi2731**
Removed the buttons from the center of the page and placed them as links in the navbar. Also removed the logo and placed it in the navbar. All the contents from the body of the main page are also removed as the calendar has to be integrated and set up with the search meeting logs bar for the recent meetings

Closes #221

### CODE REVIEW

**Comment by subhangi2731** [comment]
@darknao @t0xic0der please have a look here and let me know

File: src/mote/templates/mainpage.html [hunk 1]
```diff
@@ -32,13 +32,14 @@
         <link rel="icon" href="{{ url_for('static', filename='imgs/logoteal.png') }}" sizes="16x16">
         <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
         <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
-        <script src="{{ url_for('static', filename='jscn/socket.io.js') }}"></script>
```
Comment (darknao) [inline src/mote/templates/mainpage.html:35]: This line should not be removed.

**Review by darknao — CHANGES_REQUESTED** [review]
`imgs/Fedora_logo.png` is missing

**Comment by subhangi2731** [comment]
@darknao please check now

**Review by darknao — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**